sql/upsert_community.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE FUNCTION upsert_community() RETURNS trigger AS $$
```

Comment:
Why do we use `CREATE OR REPLACE` here but just `CREATE` for `enumerate`, above? It seems that we should have one pattern. I think I would now expect `CREATE OR REPLACE` for functions.